pkg/distributor/otlp/cmd/generate/README.md

```diff
@@ -0,0 +1,4 @@
+# OTLP translation code generator
```

Comment:
can we keep this under /tools, like all other generators (eg. doc generator)

Reply:
Sounds good.

Reply:
@pstibrany after switching to your shell script generator, I just found that it's a hassle to put the generator elsewhere in the repo tree, since pkg/distributor/otlp/timeseries.go would need to know the path relative to itself. I just put the generator script in the same directory instead. Hope you agree.

Reply:
That makes sense to me. Thank you.